j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
can we use "#!/bin/bash -e" for consistency and remove "set -ex" on line #21?

Reply:
done